<?xml version="1.0" ?>
<launch>    
    <param name="gazebo"                   value="true"    type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_op3"/>

    <param name="offset_file_path"         value="$(find diag_op3_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find diag_op3_manager)/config/OP3.robot"/>
    <param name="init_file_path"           value="$(find diag_op3_manager)/config/dxl_init_OP3.yaml"/>
    <param name="device_name"              value="/dev/ttyUSB0"/>
    
    <!-- OP3 Manager -->
    <node pkg="diag_op3_manager" type="diag_op3_manager" name="diag_op3_manager" output="screen"/>
</launch>
